spec/spec_helper.rb

```diff
@@ -5,9 +5,13 @@
 RSpec::Expectations.configuration.warn_about_potential_false_positives = false
 
 # set environment variable from .env
-lines = File.read '.env'
-lines = lines.split('\n')
-lines.each do |line|
-  line = line.split(' = ')
-  ENV[line[0]] = line[1].gsub!('"', '')
+begin
```

Comment:
maybe we shall use `if` rather than rescue
as if is 10 times faster than rescue

`if File.exist?(.env)`